src/sentry/processing/backpressure/memory.py

```diff
@@ -46,8 +46,10 @@ def query_rabbitmq_memory_usage(host: str) -> ServiceMemory:
 def get_memory_usage(node_id: str, info: Mapping[str, Any]) -> ServiceMemory:
     # or alternatively: `used_memory_rss`?
     memory_used = info.get("used_memory", 0)
-    # `maxmemory` might be 0 in development
-    memory_available = info.get("maxmemory", 0) or info["total_system_memory"]
+    # `maxmemory` might be 0 in development. We also sometimes see responses from redis
+    # that are missing keys in production, so just being defensive here and defaulting to 0
+    # rather than erroring
+    memory_available = info.get("maxmemory", 0) or info.get("total_system_memory", 0)
```

Comment:
Is it fine if this is 0 ?

Reply:
I just realised we calculate a % using this, and it'll fail